ods_ci/tests/Resources/Page/DistributedWorkloads/DistributedWorkloads.resource

```diff
@@ -53,9 +54,9 @@ Prepare Codeflare-SDK Test Setup
 
```

Comment:
`${result} =    Run Process  virtualenv -p python3.11 ${VIRTUAL_ENV_NAME}`
    It should also be updated with ${PYTHON_VERSION}

Reply:
Thanks for pointing this @ChughShilpa.. I think we created this earlier virtual env setup for 3.9 as by default venv 3.11 available in our jenkins.. will update it accordingly

Reply:
Removed virtual env creation steps as we can switch poetry env with existing virtual environment in jenkins , test successfully ran here `job/rhoai-test-flow/1109/`